packages/cryptoplease/lib/presentation/screens/authenticated/send_flow/fungible_token/confirm_screen/components/fee_view.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:cryptoplease/l10n/l10n.dart';
+import 'package:cryptoplease/presentation/screens/authenticated/send_flow/fungible_token/confirm_screen/components/list_item.dart';
+import 'package:cryptoplease_ui/cryptoplease_ui.dart';
+import 'package:flutter/material.dart';
+
+class FeeView extends StatelessWidget {
```

Comment:
Let's make this component smarter and more universal.

E.g. pass it `CryptoAmount` and `FiatCurrency` and it will render a list item with formatted amount and formatted converted currency. This way we can reuse it for both amount view and fee view (with just different styles).

Reply:
Actually both fee and amount are `CryptoAmount`s once the fee itself is also in SOL. So, I created different constructors and let the logic of converting to fiat inside the widget.